### ISSUE TITLE
Puerto Rico does not deserved the D grades

### ISSUE THREAD

**Issue by amaliagomezPR** [COVID19Tracking/issues#361]
Last report of Puerto Rico coroner office NCF (NEGOCIADO DE CIENCIAS FORENSES): "Hasta el día de hoy se han recibido 64 casos sospechosos de coronavirus para autopsias, de los cuales 53 han arrojado resultados negativos y siete están pendientes de resultado".
"To date, 64 suspected cases of coronavirus have been received for autopsy, of which 53 have yielded negative results and seven are pending results."
This is the proof the hospitals in Puerto Rico diagnose the disease without any tests; by symptoms only. But Puerto Rico suffer not only from coronavirus. Puerto Rico has the same time Dengue, influenza, Cystic fibrosis etc diseases and illnesses with the same symptoms. But influenza for example has medications which can cure patients. But hospitals in Puerto Rico did not provide influenza tests. Doctors leave the patients in hospitals died without any medical help. After patients died coroner provide coronavirus tests but not influenza or Dengue tests. 53 people was not infected by coronavirus. But coroners by autopsy did not find the reason of death of 53 people. 

Puerto Rico even does not deserved grade F. Puerto Rico kill patients in his hospitals. 

Today in Puerto Rico was registered 97 deaths. By report of coroners 64 - 53 = 11 patients had positive coronavirus tests. How and why died 97 - 11 = 86 patients? No answers. Seems to be the all records in Puerto Rico are false.

NCF office continue to use protocols from 2017 after hurricane Maria. 

https://www.primerahora.com/noticias/puerto-rico/notas/deficiente-el-manejo-del-covid-19-en-la-isla/

**Comment by quangnguyen1995** [comment]
Thank you for your comment! We have passed this issue to the states grade team.

**Comment by amaliagomezPR** [comment]
On March 2020 governor of Puerto Rico bought serological tests manufactured
by Phamatech. 50 000 tests for $13.00 each. 10 000 FDA delivered to Puerto
Rico despite serological tests by Phamatech never was approved. Puerto Rico's
Health department still continue to use this tests. If checked person
showed positive case reported by Health Department like sick case.



Please Look at the title of one of main newspaper of Puerto Rico El Nuevo
Dia:

"Salud asegura que labora en un plan para dar seguimiento a positivos
serológicos de COVID-19". "la agencia que dirige desarrollará un plan de
seguimiento a las personas que arrojan un resultado positivo en la prueba
serológica de COVID-19".

https://www.elnuevodia.com/noticias/locales/nota/saludaseguraquelaboraenunplanparadarseguimientoapositivosserologicosdecovid-19-2567125/

"Salud assures that it works on a plan to follow up serological positives
of COVID-19." "The lead agency will develop a follow-up plan for people who
test positive for the COVID-19 serology test.».



This is means all reports from Puerto Rico about situations with COVID-19
are false. The residents of Puerto Rico want to know real situations with
pandemia in Puerto Rico.



In Puerto Rico was registered 20 000 cases of influenza and about 1000
cases of Dengue. Much more casers to compare with coronavirus cases.
But in Puerto
Rico's hospitals people continue to die because of "positive serological
tests" without any medical help. Why Puerto Rico continue to kill people
supported by FDA? Maybe people which was dies suffered from influenza which
can be cured easy. Influenza, Dengue etc have very close symptoms like
coronavirus. But hospitals in Puerto Rico made diagnosis only by symptoms
and did not cure the people. Only after person dies coroner provide the
tests!
